exec-ui/src/app/modules/metrics/modules/metrics/pipeline-lead-time/strategies/pipeline-lead-time-primary-metric-strategy.ts

```diff
@@ -8,12 +8,12 @@ import {PipelineLeadTimeConfiguration} from '../pipeline-lead-time.configuration
 export class PipelineLeadTimePrimaryMetricStrategy extends PrimaryMetricStrategyBase {
   parse(counts: Count[]): MetricValueModel {
     if (!counts || !counts.length) {
-      return {name: PipelineLeadTimeConfiguration.buildingBlockLabel, value: 0, unit: 'minutes'};
+      return {name: PipelineLeadTimeConfiguration.buildingBlockLabel, value: 0, unit: 'hours'};
```

Comment:
@Sbrenthughes and @tabladrum I believe we need to stick to minutes in the metrics, What are your thoughts on this metrics change

Reply:
Reverted to minutes.